iocage/lib/Provisioning/ix.py

```diff
@@ -148,19 +178,52 @@ def provision(
     self.jail.fstab.save()
 
     if "pkgs" in pluginDefinition.keys():
-        pkg_packages = " ".join(pluginDefinition["pkgs"])
+        pkg_packages = list(pluginDefinition["pkgs"])
     else:
-        pkg_packages = pluginDefinition.name
+        pkg_packages = [pluginDefinition.name]
 
-    commands = [
-        "ASSUME_ALWAYS_YES=true pkg update",
-        f"ASSUME_ALWAYS_YES=true pkg install -y {pkg_packages}",
-        "./ix-plugin/post_install.sh"
-    ]
-
-    for event in self.jail.fork_exec("\n".join(commands)):
+    for event in self.jail.start():
         yield event
 
+    try:
+        yield packageUpdateEvent.begin()
+        try:
+            stdout, stderr, returncode = self.jail.exec(
+                ["pkg", "update"],
+                env=dict(ASSUME_ALWAYS_YES="true")
+            )
+            yield packageUpdateEvent.end(stdout=stdout)
+        except iocage.lib.errors.IocageException as e:
+            yield packageUpdateEvent.fail(e)
+            raise e
+
+        yield packageInstallEvent.begin()
+        try:
+            stdout, stderr, returncode = self.jail.exec(
+                ["pkg", "install", "-yield"] + pkg_packages,
```

Comment:
`pkg install -yield`?

Reply:
Hahahaha, damn you auto-correct. 😆(Have no other explanation) 
Of course this must be just `-y`.